### ISSUE TITLE
Query DATE_RANGE or table wildcards

### ISSUE THREAD

**Issue by juliankeppel** [spotify/spark-bigquery#42]
I want to use the BigQuery connector to load data from multiple tables at a time. I tried to use DATE_RANGE functions and table wildcards, but both doesn't work and I get the following error: `Invalid datasetAndTableString '<dataset-id>.<some-table-name-with-wildcard>'; must match regex '[a-zA-Z0-9_]+\.[a-zA-Z0-9_]+'.`

I want to load the raw data, so I can't use `sqlContext.bigQuerySelect()` and provide SQL statement, because then I would have to unnest the nested data.

tl;dr: Is it possible to load raw data from multiple tables at one time with something like DATE_RANGE or wildcard functions which are provided by BigQuery itself?
